bindings/java/src/org/sleuthkit/datamodel/OsAccount.java

```diff
@@ -0,0 +1,549 @@
+/*
+ * Sleuth Kit Data Model
+ *
+ * Copyright 2020-2021 Basis Technology Corp.
+ * Contact: carrier <at> sleuthkit <dot> org
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.sleuthkit.datamodel;
+
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+
+/**
+ * Abstracts an OS user account.
+ *
+ * An OS user account may own files and (some) artifacts.
+ *
+ */
+public final class OsAccount implements Content {
+
+	final static long NO_USER = -1;
+	final static String NULL_UID_STR = null;
```

Comment:
I found it useful when a  null value needed to be passed into a method with a long list of args, many of which are already null (example a number of deprecated methods) 

Having an explicit name doe it instead of just "null" helped me with both the placement of the arg as well as telling me what that null meant.

Reply:
That makes a lot of sense. 

I would call it NO_OWNER_ID in line with the other changes we have made and to keep it in the problem domain instead of the solution domain.

Also, NO_ACCOUNT should be null in light of other decisions.